internal/config/config.go

```diff
@@ -209,7 +211,6 @@ var SubSystemsSingleTargets = set.CreateStringSet(
 	SubnetSubSys,
 	CallhomeSubSys,
 	DriveSubSys,
-	CacheSubSys,
```

Comment:
Wrong to remove this, this is needed and valid.

Reply:
Okay, but `CacheSubSys` seems to be repeated in this list, look at lines 192 and 212, is that expected?:
https://github.com/minio/minio/blob/c1cae51fb52c4dfa3a4ad98dfc034f025d7b9b9e/internal/config/config.go#L194-L214

Reply:
Oh it's repeated then it's fine.